azureadps-2.0-preview/AzureAD/Get-AzureADDirectoryRole.md

```diff
@@ -47,7 +47,7 @@ PS C:\>Get-AzureADDirectoryRole
 
 ObjectId                             DisplayName                        Description
 --------                             -----------                        -----------
-019ea7a2-1613-47c9-81cb-20ba35b1ae48 Company Administrator              Company Administrator role has full access to perform any operation in the company scope.
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb Company Administrator              Company Administrator role has full access to perform any operation in the company scope.
```

Comment:
Same issue here.

Reply:
These look like they would be needed, as this is a role. It would break otherwise.